Dockerfile

```diff
@@ -9,6 +9,7 @@ ADD nakadi /nakadi
 RUN find ./nakadi | grep -E "(__pycache__|\.pyc|\.pyo$)" | xargs rm -rf
 
 ADD run.sh /nakadi/run.sh
+ADD scm-source.json /scm-source.json
```

Comment:
My only concern here is that it will be not possible to build docker image if the file is not present. (and it will be present only after we run build script from private repo)
Maybe it's possible to add that to container only if the file exists? (to avoid failure when the file doesn't exist)